* remove fftw_version, conflicts with the one in mkl's fftw
* http://qe-forge.org/pipermail/q-e-developers/2016-January/001088.html
author: Miguel Dias Costa (National University of Singapore)   
--- espresso-5.3.0/FFTXlib/fftw.c.orig	2016-05-26 09:54:35.911733000 +0800
+++ espresso-5.3.0/FFTXlib/fftw.c	2016-05-26 09:59:43.900568303 +0800
@@ -345,8 +345,6 @@
 #include <stdio.h>
 #include <stdlib.h>
 
-char *fftw_version = "FFTW V1.1 ($Id: fftw.c,v 1.3 2010-01-26 14:06:59 giannozz Exp $)";
-
 /*
  * This function is called in other files, so we cannot declare
  * it as static. 
--- espresso-5.3.0/FFTXlib/fftw.h.orig	2016-05-26 09:54:40.566922000 +0800
+++ espresso-5.3.0/FFTXlib/fftw.h	2016-05-26 09:59:59.822132168 +0800
@@ -133,7 +133,6 @@
 
 extern generic_codelet fftw_twiddle_generic;
 extern generic_codelet fftwi_twiddle_generic;
-extern char *fftw_version;
 
 /*****************************
  *        Plans
